### ISSUE TITLE
can't find master controls file in /etc/kube-bench/cfg: no test files found <= benchmark version: cis-1.6

### ISSUE THREAD

**Issue by revaniki** [aquasecurity/kube-bench#818]
**What steps did you take and what happened:**

Attempting to run kube-bench on a k3S cluster running via vagrant for a class I'm creating with kube-bench 😉 

Downloaded kube-bench binary locally via tar ball

On execution of `./kube-bench master` it doesn't detect the version

```bash
vagrant@master:~/tools> ./kube-bench master
unable to determine benchmark version: config file is missing 'version_mapping' section
```

hence I'm running it manually via `./kube-bench master --benchmark cis-1.6` which results in:

```bash
vagrant@master:~/tools> ./kube-bench master --benchmark cis-1.6

can't find master controls file in /etc/kube-bench/cfg: no test files found <= benchmark version: cis-1.6
```

all files are present:

```bash
vagrant@master:~/tools/cfg/cis-1.6> ll
total 88
-rwxrwxrwx 1 vagrant vagrant    77 Jan 20 13:58 config.yaml
-rwxrwxrwx 1 vagrant vagrant  1181 Jan 20 13:58 controlplane.yaml
-rwxrwxrwx 1 vagrant vagrant  4919 Jan 20 13:58 etcd.yaml
-rwxrwxrwx 1 vagrant vagrant 40638 Feb  7 23:53 master.yaml
-rwxrwxrwx 1 vagrant vagrant 20048 Jan 20 13:58 node.yaml
-rwxrwxrwx 1 vagrant vagrant  9391 Jan 20 13:58 policies.yaml
```
**What did you expect to happen:**

kube-bench should have executed and provided a result

**Environment** 

Kube-bench 0.5.0
Kubernetes Version: v1.20.2+k3s1 
Vagrant 2.2.14

**Anything else you would like to add:**

Used the work around to run kube-bench as a pod, but this adds some complexity for students learning this for the first time in a class I'm teaching.  Please take a look.

**Comment by rdbraber** [comment]
In the 'tools/cfg' directory, you need to download the main config.yaml file. It's not included in the tarball. See also #811

**Comment by yoavrotems** [comment]
Solved in #813 Thanks to @curx

**Comment by revaniki** [comment]
thanks guys @curx
